default.json

```diff
@@ -0,0 +1,45 @@
+{
+  "docker": {
+    "packageRules": [
+      {
+        "allowedVersions": "<13 || >13",
+        "packageNames": ["circleci/node", "node"]
+      },
+      {
+        "packagePatterns": ["circleci/.+"],
+        "recreateClosed": true
+      },
+      {
+        "allowedVersions": "<=3.8",
+        "packageNames": ["circleci/python", "python"]
+      },
+      {
+        "packagePatterns": ["circleci/.+"],
+        "schedule": [
+          "on Monday after 22:00",
```

Comment:
Right, this is the reason for the conflict.

Merging `recreateClosed` sounds like a good idea. 👍

Reply:
@cfra figured out this merging both objects will only recreate closed updates for digest updates. Since this is not what we intended we'll not merge them.